freyja_examples/CMakeLists.txt

```diff
@@ -31,18 +31,9 @@ include_directories(
 )
 
 # simple trajectory provider
-add_executable(trajectory_provider_node src/trajectory_provider.cpp)
-add_dependencies(trajectory_provider_node ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+add_executable(temporal_provider_node src/temporal_provider.cpp)
+add_dependencies(temporal_provider_node ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 
-target_link_libraries(trajectory_provider_node
+target_link_libraries(temporal_provider_node
    ${catkin_LIBRARIES}
- )
-
-
-# vision-based guidance for approach
-# add_executable(trajectory_shaper_node src/trajectory_shaper.cpp)
-# add_dependencies(trajectory_shaper_node ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
-
-# target_link_libraries(trajectory_shaper_node
-#   ${catkin_LIBRARIES}
-# )
+ )
```

Comment:
.. weird story here: files should end with a newline character :)
This used to be necessary for older serial terminals and file pipes, still a common practice.

Reply:
Done.

Reply:
Also didn't know that so I learnt something today. Thanks for pointing that out :)